fv/wireguard_test.go

```diff
@@ -52,7 +52,7 @@ import (
 
 const (
 	wireguardInterfaceNameDefault       = "wireguard.cali"
-	wireguardMTUDefault                 = 1420
+	wireguardMTUDefault                 = 1440
```

Comment:
Is it worth a comment to explain this value?  (And perhaps specifically why 1440 is better than 1420?)

Reply:
Good idea, just added one.